src/configuration.ts

```diff
@@ -16,19 +16,29 @@ const configuration = Object.freeze({
   },
 
   [ProviderType.WALLET_CONNECT]: {
-    urls: {
-      [ChainId.ETHEREUM_MAINNET]:
-        'https://mainnet.infura.io/v3/fa3357a65e2d4214ac735190646a3c53',
-      [ChainId.ETHEREUM_ROPSTEN]:
-        'https://ropsten.infura.io/v3/fa3357a65e2d4214ac735190646a3c53',
-      [ChainId.ETHEREUM_RINKEBY]:
-        'https://rinkeby.infura.io/v3/fa3357a65e2d4214ac735190646a3c53',
-      [ChainId.ETHEREUM_KOVAN]:
-        'https://kovan.infura.io/v3/fa3357a65e2d4214ac735190646a3c53'
-    }
+    urls: getRpcUrls()
   }
 })
 
 export function getConfiguration() {
   return configuration
 }
+
+export function getRpcUrls() {
+  return {
+    [ChainId.ETHEREUM_MAINNET]:
+      'https://mainnet.infura.io/v3/21ee2680fd58460ba96d2b3addd7c38c',
```

Comment:
Question: we were using different RPC urls for wallet connect and the `NetworkConnector`, now they'll be the same api key. Could that be a problem? maybe in terms of quota

Reply:
I'll check out the reason

Reply:
Updated RPCS for wallet connect